website/versioned_docs/version-25.x/MockFunctions.md

```diff
@@ -69,12 +77,18 @@ expect(someMockFunction.mock.calls[0][1]).toBe('second arg');
 // The return value of the first call to the function was 'return value'
 expect(someMockFunction.mock.results[0].value).toBe('return value');
 
+// The function was called with a certain `this` context: the `element` object.
+expect(someMockFunction.mock.contexts[0]).toBe(element);
+
```

Comment:
Hm.. I guess these are accidental changes. Jest v25 does not have `mock.contexts` it was introduced recently.